SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialClassInfoManager.h

```diff
@@ -110,4 +110,6 @@ class SPATIALGDK_API USpatialClassInfoManager : public UObject
 	TMap<Worker_ComponentId, TSharedRef<FClassInfo>> ComponentToClassInfoMap;
 	TMap<Worker_ComponentId, uint32> ComponentToOffsetMap;
 	TMap<Worker_ComponentId, ESchemaComponentType> ComponentToCategoryMap;
+
+	void QuitGame();
```

Comment:
nit: there's a private functions section in this file, could you move this definition there